src/Select/Select.tsx

```diff
@@ -50,7 +50,14 @@ const StyledSelect = styled.select`
 `
 
 const ArrowIndicatorSVG: React.FC<React.PropsWithChildren<{className?: string}>> = ({className}) => (
-  <svg width="16" height="16" fill="currentColor" xmlns="http://www.w3.org/2000/svg" className={className}>
+  <svg
```

Comment:
Not related to the PR changes - Interesting to see we use SVG here not Octicons. I wonder if I miss any context here 🤔

Reply:
Same 🤔 I'll post a message about it in Slack to see what folks say.